kedro/templates/project/hooks/utils.py

```diff
@@ -172,6 +198,10 @@ def setup_template_tools(selected_tools_list: str, requirements_file_path: str,
 
     if ("PySpark" in selected_tools_list or "Kedro Viz" in selected_tools_list) and example_pipeline != "True":
         _remove_pyspark_viz_starter_files("Kedro Viz" in selected_tools_list, python_package_name)
+        # Remove requirements used by example pipelines
+        _remove_from_file(requirements_file_path, example_pipeline_requirements)
+        _remove_extras_from_kedro_datasets(requirements_file_path)
```

Comment:
We also need to do this when there's no example chosen, but also no `pyspark` or viz right? https://github.com/kedro-org/kedro-starters/blob/main/spaceflights-pandas/%7B%7B%20cookiecutter.repo_name%20%7D%7D/requirements.txt

Reply:
But `scikit-learn` is also a requirement in `spaceflights-pandas` as I've linked above.

Reply:
Ah yes you're right!